spec/translator_spec.lua

```diff
@@ -14,11 +15,58 @@ local function assert_translation(pallene_code, expected)
     assert.are.same(expected, contents)
 end
 
+local function assert_translation_error(pallene_code, expected)
+    assert(util.set_file_contents("__test__.pln", pallene_code))
+    local ok, _, _, actual = util.outputs_of_execute("./pallenec __test__.pln --emit-lua")
+    assert.is_false(ok)
+    assert.match(expected, actual, 1, true)
+end
+
 local function cleanup()
     os.remove("__test__.pln")
     os.remove("__test__.lua")
 end
 
+describe("Syntax errors during Pallene to Lua translation", function ()
```

Comment:
We could replace each of these describe blocks with a single test, as discussed on Slack.